src/blend/blend.rs

```diff
@@ -28,12 +28,13 @@ pub trait Blend: Sized {
     ///```
     ///use palette::{Rgb, Rgba, Blend};
     ///use palette::blend::PreAlpha;
+    ///use palette::white_point::D65;
     ///
-    ///type PreRgba = PreAlpha<Rgb<f32>, f32>;
+    ///type PreRgba = PreAlpha<Rgb<D65, f32>, f32>;
     ///
     ///fn blend_mode(a: PreRgba, b: PreRgba) -> PreRgba {
     ///    PreAlpha {
-    ///        color: Rgb::new(a.red * b.green, a.green * b.blue, a.blue * b.red),
+    ///        color: Rgb::with_wp(a.red * b.green, a.green * b.blue, a.blue * b.red),
```

Comment:
There was a reason I changed it. I can't figure out why with out the tests working :)